### ISSUE TITLE
Toolbar buttons are too wide

### ISSUE THREAD

**Issue by marhkb** [rafaelmardojai/firefox-gnome-theme#218]
The buttons in the "toolbar" seem to be too wide. See the attached picture that compares them with epiphany.
Epiphany's buttons have dimensions of (36, 34), wheras firefox's buttons have dimensions of (38, 34). 

I am not sure whether this is a regression introduced in firefox 88 or whether this was already present in earlier versions (at least I didn't notice).

I am on current master, btw.

![too_wide_buttons](https://user-images.githubusercontent.com/3630213/115990605-7335a280-a5c4-11eb-812a-bfdbfd0115a3.png)

**Comment by rafaelmardojai** [comment]
What Firefox version are you using?

**Comment by marhkb** [comment]
I am on firefox 88.0 from Arch Linux.
The only thing I changed in order to get bright icons with papirus icon theme was

```
diff --git a/theme/system-icons.css b/theme/system-icons.css
index f8ed0be..2c64417 100644
--- a/theme/system-icons.css
+++ b/theme/system-icons.css
@@ -7,8 +7,8 @@
 }
 @media (prefers-color-scheme: dark) {
        :root {
-               --gnome-icons-hack-filter: invert(100%) sepia(100%) grayscale(100%) brightness(200%) brightness(200%);
-               --gnome-window-icons-hack-filter: invert(90%);
+               /*--gnome-icons-hack-filter: invert(100%) sepia(100%) grayscale(100%) brightness(200%) brightness(200%);*/
+               /*--gnome-window-icons-hack-filter: invert(90%);*/
        }
 }
```

**Comment by rafaelmardojai** [comment]
Hmm strange..  could you try the `beta` branch? It has some fixes for a similar issue in `89`.

**Comment by marhkb** [comment]
I've just tested on my other computer. And It is working there. The only differenence is that the resolution there is 1920x1080.
This computer on the other hand has a resolution of 2560x1440. I have just figured out that if I set firefox "density" mode to "compact" the buttons have correct dimensions. Maybe firefox treats the higher resolution in a different way. 
https://www.youtube.com/watch?v=mJbc_oRJrKM

![compact](https://user-images.githubusercontent.com/3630213/116005713-81a5ad80-a608-11eb-81de-8b5bdc00b610.png)

I think this can be closed.
